Task3/MoneyManager/MoneyManager.DataAccess/ApplicationContext.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Linq;
+using  Microsoft.EntityFrameworkCore;
+using MoneyManager.DataAccess.Models;
+
+namespace MoneyManager.DataAccess
+{
+    public class ApplicationContext : DbContext, IApplicationContext
+    {
+
+        public ApplicationContext(DbContextOptions<ApplicationContext> options) : base(options)
+        {
+            Database.EnsureCreated();
+        }
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
+            modelBuilder.Entity<User>().ToTable("Users");
```

Comment:
Why do you specify `ToTable` here? Usually DB table name is named in singular (not plural).

Reply:
You don't have to use `ToTable` here at all. EF automatically names tables according to the class name.